### DISCUSSION TITLE
Webinstaller not working for me.

### DISCUSSION THREAD

**Discussion by ulltieri** [jomjol/AI-on-the-edge-device#2713] [Q&A]
Hello!
I can´t get the webinstaller to work. It finds my ESP32-CAM and after pressing install it get stuck on "Installing prepairing instalation". When I try the same device and hookup on "ESP Web Tools" page it works with no problems.
Any ideas?
//Erik

**Comment by caco3** [comment]
You are the first who mentions such issue, thus it is difficult for us to understand it.
Can you try it with a different (chrome-based) browser?
Or another device?
Did you press the reset/boot button at the right time?

Alternatively there is still the manual way to set a device up!

**Reply by cdekesel** [reply]
I have the same issue, using ESP32-CAM with a ESP32-CAM-MB adapter.
Tried with different browser (Chrome, Edge)
Tried with different ESP32-CAM devices.

What is the right time or sequence for pressing the BOOT/RESET button(s)

**Reply by SybexX** [reply]
![Unbenannt](https://github.com/jomjol/AI-on-the-edge-device/assets/587201/7fa2929a-4759-4317-be97-3e4dd6b04190)
